lib/pure/strutils.nim

```diff
@@ -189,6 +189,33 @@ proc isUpper*(s: string): bool {.noSideEffect, procvar,
   for c in s:
     result = c.isUpper() and result
 
+proc isTitle*(s: string): bool {.noSideEffect, procvar,
```

Comment:
Disregard all of that. I misunderstood how unicode works in Nim. I'll switch this over to the unicode module along with `title`.